plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/controls/querylog/QueryLogViewer.java

```diff
@@ -1082,8 +1082,16 @@ protected Composite createDialogArea(Composite parent) {
             messageLabel.setLayoutData(new GridData(GridData.VERTICAL_ALIGN_BEGINNING));
 
             Control msg;
-            if (object.getObject() instanceof QMMStatementExecuteInfo) {
+            if (object.getObject() instanceof QMMStatementExecuteInfo qmmStatementExecuteInfo) {
                 msg = createSQLPanel(topFrame);
+                Composite sqlDetailsPanel = UIUtils.createPlaceholder(topFrame, 4, 5);
```

Comment:
I don't like that labels are misaligned. Let's wrap catalog and schema names so they are each on a separate line.

Reply:
Use createComposite()